system_test/aest_sync_SUITE.erl

```diff
@@ -428,8 +428,8 @@ add_spend_txs(Node, SenderAcct, N, NonceStart) ->
 
 add_spend_tx(Node, #{ pubkey := SendPubKey, privkey := SendSecKey }, Nonce) ->
     #{ public := RecvPubKey, secret := RecvSecKey } = enacl:sign_keypair(),
-    Params = #{ sender => SendPubKey
-              , recipient => RecvPubKey
+    Params = #{ sender => aec_id:create(account, SendPubKey)
```

Comment:
It looks like this was cherry-picked from master and squashed - rather than rebased on master. Ok anyway.

Reply:
Not really cherry-picked. I rebased and for some reason, something went weird and it wasn't there... I had to add it back... sorry about that.